microsoft/PowerToys
# [Shortcut Guide] Save settings to file instead of sending to the runner

## Summary of the Pull Request

**What is this about:**

**What is include in the PR:** 

**How does someone test / validate:** 
Change Shortcut guide settings and verify that changes are populated to the settings file
 
## Quality Checklist

- [X] **Linked issue:** #11078

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
